MSAL/src/public/MSALSilentTokenParameters.h

```diff
@@ -42,6 +42,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic) BOOL forceRefresh;
 
+/**
+ 1. When broker is presenting on the device
+    Default is false. when broker failed to return a (new) access token, ignores existing refresh token in local cache, and return broker error.
+    If set to true, when broker failed to return a (new) access token, tries with existing refresh token in the cache, and return results.
+ 2. When broker is not presenting on the device
```

Comment:
How do we know that broker is available? Does this applies only to sso ext? If yes, should we replace the word "broker" with "SSO ext"?

Reply:
Yes, for silent flow, this refers to Sso Ext

Reply:
Updated